src/OrchardCore/OrchardCore.OpenId.Core/YesSql/Helpers/OpenIdExtensions.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Collections.Generic;
+
+namespace YesSql
+{
+    public static class OpenIdExtensions
+    {
+        internal static IAsyncEnumerable<T> AsAsyncEnumerable<T>(this IQuery<T> query) where T : class
```

Comment:
@sebastienros it's something that would be really useful to have in YesSql RTM, as it will be massively used in 3.0.